src/modules/service-catalog/hooks/useItemFormFields.tsx

```diff
@@ -5,6 +5,8 @@ import type { TicketFieldObject } from "../../ticket-fields/data-types/TicketFie
 import type { EndUserCondition } from "../../ticket-fields/data-types/EndUserCondition";
 import { getCustomObjectKey } from "../../ticket-fields/fields/LookupField";
 import { getVisibleFields } from "../../ticket-fields/getVisibleFields";
+import DOMPurify from "dompurify";
+import { ALLOWED_HTML_TAGS } from "../../new-request-form/constants";
```

Comment:
This will couple both applications which is not something we'd want to do.
As a rule of thumb we never want this and applications should not inherit from each-other.